charts/invenio/templates/logstash/filebeat-configmap.yaml

```diff
@@ -4,6 +4,8 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: filebeat-config
+  labels:
```

Comment:
I like that you try to fix the label situation, but I think this belongs in another PR. It's hard to review multiple things in one PR on GitHub.

Reply:
But again: I _really_ appreciate how good you are at creating separate commits for each change. It's so nice to see someone that creates atomic commits for a change! ❤️

Reply:
I know; it's just faster for me to work like that sometimes. I'm making some of the changes because I feel they need to be done, not because we actually need them.

On the `Files changed` tab, you can select the commit you want to check. That may help.

Reply:
Also, I just realized that I based this one on top of the other PR ... that's probably also confusing 😂

Reply:
Yeah, the changes that are from other PRs must for sure be removed here, very hard to review otherwise. 🙂

Reply:
Done, I moved the label stuff [here](https://github.com/inveniosoftware/helm-invenio/pull/138) and rebase.